charts/frr-k8s/templates/controller.yaml

```diff
@@ -188,7 +188,7 @@ spec:
         args:
         - "--node-name=$(NODE_NAME)"
         - "--namespace=$(NAMESPACE)"
-        - "--metrics-bind-address=127.0.0.1:{{ .Values.prometheus.metricsPort }}"
+        - "--metrics-bind-address=:{{ .Values.prometheus.metricsPort }}"
```

Comment:
I think binding to 127.0.0.1 is better than listening on any

Reply:
I changed it here so it is aligned with the regular manifests and the code's default (when we generated the controller), do you think we should change it everywhere?

Reply:
yeah I think this is better as the pod is hostnetworked adn we won't expose the port outside.